bluetooth-CC2650-demo/README.md

```diff
@@ -0,0 +1,115 @@
+# Bluetooth With KubeEdge Using CC2650
+
+
+Users can make use of KubeEdge platform to connect and control their bluetooth devices, provided, the user is aware of the of the data sheet information for their device.
+Kubernetes Custom Resource Definition (CRD) and KubeEdge bluetooth mapper is being used to support this feature, using which users can control their device from the cloud. Texas Instruments CC2650 SensorTag device is being shown here as an example.
+
+
+## Description
+
+KubeEdge support for bluetooth protocol has been demonstrated here by making use of Texas Instruments CC2650 SensorTag device.
+This section contains instructions on how to make use of bluetooth mapper of kubeedge to control CC2650 SensorTag device.  
+  
+  We will only be focusing on the following features of CC2650 :-
+  
+  ```shell
+  1. IR Temperature
+  2. IO-Control :
+      2.1 Red Light
+      2.2 Greem Light
+      2.3 Buzzer
+      2.4 Red Light with Buzzer
+      2.5 Green Light with Buzzer
+      2.6 Red Light along with Green Light
+      2.7 Red Light, Green Light along with Buzzer 
+      
+  ```
+  
+  The bluetooth mapper has the following major components :-
+   - Action Manager
+   - Scheduler
+   - Watcher
+   - Controller
+   - Data Converter
+  
+  Please click [Bluetooth_Mapper](https://github.com/kubeedge/kubeedge/blob/master/device/bluetooth_mapper/README.md) to understand the features and  the usage of bluetooth mapper.
+  
+  
+## Prerequisites 
+
+### Hardware Prerequisites
+
+1. Texas instruments CC2650 bluetooth device
+2. Linux based edge node with bluetooth support  (An Ubuntu 18.04 laptop has been used in this demo) 
+
+### Software Prerequisites
+ 
+1. Golang (Version 1.11.4 has been used for this demo)
+2. KubeEdge (Version 0.3 has been used for this demo)
+
+## Steps to reproduce
+
+1. Clone and run KubeEdge. 
+    Please click [KubeEdge Usage](https://github.com/kubeedge/kubeedge/blob/master/docs/getting-started/usage.md) for instructions on the usage of KubeEdge.
+    Please ensure that the kubeedge setup is up and running before execution of step 4 (mentioned below).
+
+2. Clone the kubeedge/examples repository.
+
+       ```shell
+                   git clone https://github.com/kubeedge/examples.git $GOPATH/src/github.com/kubeedge/examples
+        ```
+
+3. Create the CC2650 SensorTag device model and device instance.
+
+       ```shell
+                   cd $GOPATH/src/github.com/kubeedge/examples/bluetooth-CC2650-demo/sample-crds
+                   kubectl apply -f CC2650-device-model.yaml
+                   kubectl apply -f CC2650-device-instance.yaml
+
+                   Note: You can change the CRDs to match your requirement
+        ```
+ 
+4. Please ensure that bluetooth service of your device is ON
+
+5. Set 'bluetooth=true' label for the node (This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node)
```

Comment:
can we add more details , like hardware / software stack supported node can be only used to deploy etc ?

Reply:
The required hardware and software stack required has been mentioned in the Prerequisites section. Is that sufficient or should it be repeated here again?

Reply:
This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node  -->

This label is a prerequisite for the scheduler to schedule bluetooth_mapper pod on the node  [  which meets the hardware / software prerequisites ]

Reply:
Done, thanks